### PR TITLE
feat(launch): Rename Local Runners Continued

### PR DESCRIPTION
**Pull request by hu-po**
Fixes WB-NNNN

Description
-----------
Ties up some loose ends for the rename of Local Runners (LocalContainer and LocalProcess).

Testing
-------
How was this PR tested?

Checklist
-------
- Include reference to internal ticket "Fixes WB-NNNN" (and github issue "Fixes #NNNN" if applicable)

### CODE REVIEW

**Review by KyleGoyette — COMMENTED** [review]
test_launch_cli.py and test_launch.py will both need a find replace on `runner.local` to `runner.local_container`.

**Review by hu-po — COMMENTED** [review]
changed

File: wandb/sdk/launch/utils.py [hunk 1]
```diff
@@ -183,7 +183,11 @@ def parse_wandb_uri(uri: str) -> Tuple[str, str, str]:
     stripped_uri = re.sub(
         _WANDB_QA_URI_REGEX, "", stripped_uri
     )  # also for testing just run it twice
-    entity, project, _, name = stripped_uri.split("/")[1:]
+    try:
+        entity, project, _, name = stripped_uri.split("/")[1:]
+    except ValueError as e:
```
Comment (KyleGoyette) [inline wandb/sdk/launch/utils.py:188]: Why not just raise an exception here?

**Review by KyleGoyette — COMMENTED** [review]
Looks okay, I dont understand why we're changing parse to return empty strings. Shouldn't it just throw an error if we pass a non wandb uri to parse_wandb_uri, that means a path has failed.

File: wandb/sdk/launch/utils.py (hunk 1 above)
Comment (KyleGoyette) [reply wandb/sdk/launch/utils.py:188]: Fantastic

**Pull request merged**